templates/components/agents/typescript/financial_report/app/api/chat/workflow/finReport.ts

```
@@ -0,0 +1,274 @@
+   * Analyze a research result or a tool call for code interpreter from the LLM
+   */
+  private async handleAnalyze(ctx: Context, ev: AnalyzeEvent) {
+    ctx.writeEventToStream(
+      new AgentRunEvent({
+        name: "Analyst",
+        text: `Starting analysis`,
+      }),
+    );
+    // Request by workflow LLM, input is a list of tool calls
+    let toolCalls: ToolCall[] = [];
+    if (ev.data.input instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)